README.md

```diff
@@ -1,10 +1,56 @@
-# opm-parser ![alt text](https://travis-ci.org/OPM/opm-parser.svg?branch=master "TravisCI Build Status")
+# opm-parser [![Build Status](https://travis-ci.org/OPM/opm-parser.svg?branch=master "TravisCI Build Status")](https://travis-ci.com/OPM/opm-parser)
 
-### eclipse
+opm-parser is a small library for parsing ECLIPSE datafiles.
 
-A small library for parsing the ECLIPSE datafile.
 
-Dependencies:
--  boost version 1.45 or newer. If cmake does not find it, specify the
-   boost root when running cmake, like this:  
-   `cmake -DBOOST_ROOT=/path/to/boost path/to/project`
+## Dependencies
+
+- [boost](http://www.boost.org) version 1.45 or newer.
+- [libecl](https://github.com/Statoil/libecl) 2.2 or newer.
+
+
+## Build instructions
+
+We assume `boost` is installed in `BOOST_INCLUDE=/usr/local` and that you
+wish to install opm-parser in `INSTALL_PATH=/usr/local`.
+
+
+Obtaining the data:
+
+```
+git clone https://github.com/Statoil/libecl
+git clone https://github.com/OPM/opm-parser
+```
+
+
+Configure the paths:
+
+```
+export INSTALL_PATH=/usr/local
```

Comment:
Nice - since the instructions are general - and apply to an arbitrary path, it would maybe be better to use e.g `/opt/software` or something else totally non-standard for these environment variables; the all-default `/usr/local` based install should Just Work&trade;.

Reply:
Updated.

Reply:
Cool